### DISCUSSION TITLE
More detailed instructions for Linux server installation needed

### DISCUSSION THREAD

**Discussion by ghost** [thearyadev/MSRF#35] [Q&A]
Hello, first of all thank you for creating this. I wanted to ask if it is possible to install this on a ARM64 Debian server (I have a Raspberry Pi). My plan was to have this running on my Raspberry Pi and access it through the web browser. 

Thanks in advance.

**Comment by thearyadev** [comment]
the docker image currently doesn't have multiarch support. 

I'll look into arm/multiarch support. It's not something I've done before, but ill keep you updated on my progress.
